### DISCUSSION TITLE
csv file from GC -FID will only import as FTIR (and is empty)

### DISCUSSION THREAD

**Discussion by afluffycrow** [OpenChrom/openchrom#472] [Q&A] [answered]
I have reformatted a file as per the wiki csv entry and am attempting to load it into openchrom. Through the Data Explorer, when I Open Selected Measurements an empty graph with the axes "Light measured" vs "mirror position" appears. The only option on right click other than reloading the folder is "open as FTIR (.csv)".
My CSV file does not appear when I attempt File/Open Chromatogram.
I have not been able to get any of the import wizards to open the file.
Some said "there is a problem with opening the chromatogram".
Is there an import method that I have not found or a problem with my file (attached)?
[150517-135-60-0.9 openchrom.csv](https://github.com/OpenChrom/openchrom/files/8342918/150517-135-60-0.9.openchrom.csv)

**Comment by afluffycrow** [comment]
Note, this is GC data sent by a collaborator. It does not have an MS m/z component so I added 1 as the m/z for the relative abundance column.

**Comment by Mailaender** [comment, marked as answer]
Instead of shoehorning FID data using [our crude .csv format](https://github.com/OpenChrom/openchrom/wiki/csv) which was designed for GC-MS try the following:

```tab
7.98	0.000133
```

Separate RT and abundance with a tab and save the file as `.xy`.

**Comment by afluffycrow** [comment]
Is it even possible to still create .xy files? Isn't that a non supported file format in windows?

**Comment by Mailaender** [comment]
I can't quite follow. It is an extension recognized by OpenChrom. It shouldn't matter if Windows supports it. See https://www.techadvisor.com/how-to/windows/windows-10-file-extensions-3697651/ on how to see and then be allowed to change file endings.

**Comment by afluffycrow** [comment]
Thanks, this is resolved. I am able to save as .xy through notepad and the imports are now possible.
